python/pyspark/cloudpickle/cloudpickle_fast.py

```diff
@@ -556,7 +556,7 @@ def dump(self, obj):
         # `dispatch` attribute.  Earlier versions of the protocol 5 CloudPickler
         # used `CloudPickler.dispatch` as a class-level attribute storing all
         # reducers implemented by cloudpickle, but the attribute name was not a
-        # great choice given the meaning of `Cloudpickler.dispatch` when
+        # great choice given the meaning of `Cloudpickle.dispatch` when
```

Comment:
?

Reply:
Hmmm, looks like it should be `CloudPickler.dispatch` and not `Cloudpickler.dispatch`..